frontend/packages/dev-console/src/utils/pipeline-utils.ts

```diff
@@ -290,3 +290,21 @@ export const constructCurrentPipeline = (
     status,
   };
 };
+
+export const pipelineRunDuration = (run: PipelineRun): string => {
+  const startTime = _.get(run, ['status', 'startTime'], null);
+  const completionTime = _.get(run, ['status', 'completionTime'], null);
+
+  // Duration cannot be computed if start time is missing or a completed/failed pipeline has no end time
+  if (!startTime || (!completionTime && pipelineRunFilterReducer(run) !== 'Running')) {
```

Comment:
Recommend we start using `pipelineRunStatus` which is what `pipelineRunFilterReducer` calls when we want just the status value (and not a pretty display value). Obviously it doesn't matter in this case, but it doesn't hurt to start the practice in places where we want just the data.

Reply:
Changing this to use PipelineRunStatus